tensorwaves/physics/__init__.py

```diff
@@ -1 +1,34 @@
+"""Evaluateable physics models for amplitude analysis."""
+
+__all__ = ["helicityformalism"]
+
+import logging
+
+import yaml
+
 from . import helicityformalism
+
+
+def load_particle_list(particle_list_file: str) -> dict:
```

Comment:
All particle functionality may have to be migrated to the [expert system](https://github.com/ComPWA/expertsystem)

Reply:
Yes, I think once the expert system is compatible with tensorwaves we can migrate this code